cmd/bootstrapper/main.go

```diff
@@ -24,6 +24,8 @@ var (
 
 func init() {
 	rootCmd.PersistentFlags().AddGoFlagSet(flag.CommandLine)
+	// Set up logger that sends logs to StdErr. Info level logs should be bubbled up to StdOut instead
+	// WMC interprets logs in StdErr as an indication that bootstrapping failed
```

Comment:
s/WMC/WMCO

Please call why we are doing this. Namely repeat what you are saying in the commit message here too so that folks who try to change this in the future are aware of the issue they will run into without running into the issue and then having to go through the same debugging process (or git forensics) you went through to figure out why it is happening.